FFXIVClientStructs/FFXIV/Component/GUI/AtkTextNode.cs

```diff
@@ -78,6 +78,9 @@ public unsafe partial struct AtkTextNode : ICreatable
     [MemberFunction("E8 ?? ?? ?? ?? 8D 4E 32")]
     public partial void SetText(byte* str);
 
+    [MemberFunction("E8 ?? ?? ?? ?? 80 38 00 74 27")]
+    public partial Utf8String GetString();
```

Comment:
This returns a `byte*` (specifically the StringPtr of the Utf8String at `a1 + 0xB8`).

Reply:
Can't use [GenerateCStrOverloads] for `byte*` returns, you use that for functions that take `byte*` as input arguments

Reply:
I know. I've edited the comment (to specify SetText), because I didn't understand GitHubs UI, lol.